### PR TITLE
Complete PDF download/viewing functionality

### PR DESCRIPTION
**Pull request by cmirza**
- Added 'case_url' column to the cases table and added relevant seed data.
- Updated uploadRouter.js to add UUID filename for uploaded PDF and stored in case_url on case record in DB
- Removed comment from AWS.js that referenced inaccurate convention for filename

Also, the `.env` file needs to be updated for this to work:

`AWS_ACCESS_KEY` needs to be changed to `AWS_ACCESS_KEY_ID`
`AWS_SECRET_KEY` needs to be changed to `AWS_SECRET_ACCESS_KEY`

### CODE REVIEW

File: api/upload/uploadRouter.js [hunk 1]
```diff
@@ -42,16 +42,17 @@ const uploadFile = (fileName) => {
       throw err;
     }
     console.log(`File uploaded successfully. ${data.Location}`);
+
     axios
       .post(`${process.env.DS_API}${fileName}`, { name: fileName })
       .then((res) => {
-        console.log('AwwwwWWW YASSSSS 👁️👄👁️ ');
```
Comment (JusticeMatthew) [inline api/upload/uploadRouter.js:48]: Nice remembering to remove this as Ryan said 👍

**Review by JusticeMatthew — APPROVED** [review]
Looks good!

It looks like all the seed files have the same UUID, may need to address that in the future

File: api/upload/uploadRouter.js (hunk 1 above)
Comment (c-melchor) [reply api/upload/uploadRouter.js:48]: It seems like these changes are breaking the GET endpoints for judges and cases. We might have to refactor those when this is merged.

Comment (JusticeMatthew) [reply api/upload/uploadRouter.js:48]: Weird that it's affecting those given what files he worked in 🤔  hmmmm much to consider there is

Comment (cmirza) [reply api/upload/uploadRouter.js:48]: I'm not able to replicate the issue with a broken GET on cases (ex. http://localhost:8080/case/2/). I can replicate the broken get on judges (ex. http://localhost:8080/judge/David%20W.%20Crosland). The error is a bad SQL statement in judgeModel.js (error is "message": "select * from \"cases\" where \"judge_name\" = $1 - column \"judge_name\" does not exist"). This issue seems unrelated to my changes as I can replicate it on the deployed site (https://a.humanrightsfirstasylum.dev/judge/David%20W.%20Crosland).

Comment (c-melchor) [reply api/upload/uploadRouter.js:48]: Since we have realized the issues are unrelated to PR, updating my review to approve this PR.

**Review by c-melchor — APPROVED** [review]

**Pull request merged**